README.md

```diff
@@ -166,3 +166,4 @@ The example application is divided into four services which perform actions on a
 
 ## License
 This project is licensed under [Eclipse Public License - v 2.0](LICENSE). 
+
```

Comment:
> you added a licence header to the readme, but wasn't the sample guidance we followed to only add the licence file into the project and other references weren't necessary?

I didn't add the license. We can remove it, but I wonder how many other projects have this...